From 1c16c6aab3d62d5ff0b83eebbd7b20372f1de389 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E6=89=8B=E7=93=9C=E4=B8=80=E5=8D=81=E9=9B=AA?=
 <nanaeonn@outlook.com>
Date: Thu, 23 Oct 2025 14:29:59 +0800
Subject: [PATCH 7/7] Update allformats.c

---
 libavformat/allformats.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/libavformat/allformats.c b/libavformat/allformats.c
index 305fa46532..8ea2860501 100644
--- a/libavformat/allformats.c
+++ b/libavformat/allformats.c
@@ -576,6 +576,10 @@ extern const FFInputFormat  ff_libmodplug_demuxer;
 extern const FFInputFormat  ff_libopenmpt_demuxer;
 extern const FFInputFormat  ff_vapoursynth_demuxer;
 
+/* SILK */
+extern const FFInputFormat  ff_silk_demuxer;
+extern const FFOutputFormat ff_silk_muxer;
+
 #include "libavformat/muxer_list.c"
 #include "libavformat/demuxer_list.c"
 
-- 
2.47.0.windows.2

