From 69c753ff952b1c03c369b7b8461332c8a76e0567 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E6=89=8B=E7=93=9C=E4=B8=80=E5=8D=81=E9=9B=AA?=
 <nanaeonn@outlook.com>
Date: Thu, 23 Oct 2025 09:40:17 +0800
Subject: [PATCH 12/12] update

---
 libavcodec/Makefile | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/libavcodec/Makefile b/libavcodec/Makefile
index 242107b960..16c03f4bb8 100644
--- a/libavcodec/Makefile
+++ b/libavcodec/Makefile
@@ -184,9 +184,7 @@ OBJS-$(CONFIG_WMA_FREQS)               += wma_freqs.o
 OBJS-$(CONFIG_WMV2DSP)                 += wmv2dsp.o
 
 # decoders/encoders
-OBJS-$(CONFIG_SILK_DECODER)              += silkdec.o \
-                                           silk_v3_decoder/src/*.c \
-                                           silk_v3_decoder/src/*.S
+OBJS-$(CONFIG_SILK_DECODER)              += silkdec.o
 OBJS-$(CONFIG_SILK_ENCODER)              += silkenc.o
 OBJS-$(CONFIG_ZERO12V_DECODER)         += 012v.o
 OBJS-$(CONFIG_A64MULTI_ENCODER)        += a64multienc.o elbg.o
-- 
2.47.0.windows.2

